lib/generators/react_on_rails/bootstrap_generator.rb

```diff
@@ -20,7 +20,12 @@ def append_to_assets_initializer
 
           Rails.application.config.assets.precompile += %w( generated/server-bundle.js )
         DATA
-        append_to_file("config/initializers/assets.rb", data)
+        assets_intializer = File.join(destination_root, "config/initializers/assets.rb")
+        if File.exist?(assets_intializer)
+          append_to_file(assets_intializer, data)
+        else
+          puts_setup_file_error(assets_intializer, data)
```

Comment:
:+1: I'll update this to use `create_file(assets_intializer, data)` and I'll also add some test.